src/plugins/workspace/server/plugin.ts

```diff
@@ -115,12 +116,44 @@ export class WorkspacePlugin implements Plugin<WorkspacePluginSetup, WorkspacePl
       if (path === '/') {
         const workspaceListResponse = await this.client?.list(
           { request, logger: this.logger },
-          { page: 1, perPage: 1 }
+          { page: 1, perPage: 100 }
```

Comment:
```suggestion
          { page: 1, perPage: 2 }
```
I guess 2 should be good enough?